samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -116,3 +116,4 @@ CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=n
 CONFIG_NRF_CLOUD_ALERT=y
 CONFIG_NRF_CLOUD_LOG_DIRECT=y
 CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL=3
+CONFIG_TEST_COUNTER=y
```

Comment:
I have no problem with this, but is this intentional?

Reply:
Is what intentional?  Enabling the test counter?  Why is it disabled?

I can remove it, but wondering why it is the way it is.

Reply:
When I originally added it, I conceived of it as an assistance feature for testing that the sample itself works, rather than as a demo feature, but I guess you could also view it as a demo feature, in which case changing the KConfig itself to be enabled by default might make sense